hw/drivers/led/lp5523/src/lp5523.c

```diff
@@ -635,20 +634,32 @@ lp5523_get_engine_mapping(struct led_itf *itf, uint8_t engine,
 static int
 lp5523_set_pr_instruction(struct led_itf *itf, uint8_t addr, uint16_t *ins)
 {
-    uint8_t mem[2] = { ((*ins) >> 8) & 0x00ff, (*ins) & 0x00ff };
+    int rc;
+
+    uint8_t byte1[1] = { ((*ins) >> 8) & 0x00ff };
```

Comment:
changing mem[2] to byte[1] byte2[1] is a little bit extreme and does improve anything I guess

Reply:
@kasjer you mean doesn't, correct ?